src/main/resources/templates/create-user.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html PUBLIC "http://www.thymeleaf.org/dtd/xhtml1-strict-thymelefa-4.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml"
+      xmlns:th="http://www.thymeleaf.org">
+<head>
+    <title>Finance Friend Login</title>
```

Comment:
Can you change the title to something that indicates this is where the user creates their account?